README.md

```diff
@@ -501,6 +501,55 @@ The IP configuration supports the following options:
     Note that Classless inter-domain routing (CIDR) notation or network mask notation
     are not supported yet.
 
+- `routing_rule`
+
+    The policy routing rules can be specified via a list of rules given in the
+    `routing_rule` option, which allow routing the packets on other packet fields
+    except for destination address. The default value is a an empty list. Each rule is
+    a dictionary with the following entries:
+   - `priority` -
+      The priority of the rule. A valid priority ranges from 0 to 4294967295. Higher
+      number means lower priority.
+   - `action` -
+      The action of the rule. The possible values are `to-table` (default),
+      `blackhole`, `prohibit`, `unreachable`.
+   - `dport`-
+      The range of the destination port. The range consists of start and end, which can
+      be specified in the form of `$start - $end`. The valid range value has to falls
+      within the range `0 - 65534`.
+   - `family` -
+      The IP family of the rule. The possible values are `ipv4`, `ipv6`.
+   - `from` -
+      The source address of the packet to match (e.g. `192.168.100.58/24`).
+   - `fwmark` -
+      The fwmark value of the packet to match.
+   - `fwmask` -
+      The fwmask value of the packet to match.
+   - `iif` -
+      Select the incoming interface name to match.
+   - `invert` -
+      Invert the selected match of the rule.
+   - `ipproto` -
+      Select the IP protocol value to match, the valid value ranges from 1 to 255.
+   - `oif` -
+      Select the outgoing interface name to match.
+   - `sport` -
+      The range of the source port. The range consists of start and end, which can
+      be specified in the form of `$start - $end`. The valid range value has to falls
+      within the range `0 - 65534`.
+   - `suppress_prefixlength` -
+      Reject routing decisions that have a prefix length of the specified or less.
+   - `table` -
+      The route table to look up for the `to-table` action.
```

Comment:
What is the to-table action?

Reply:
The role currently supports the action `to-table`(default), `blackhole`, `prohibit`, `unreachable`.  

About the `to-table` action, I refered from kernel and NM, which basically stands for table lookup:
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/include/uapi/linux/fib_rules.h?id=f443e374ae131c168a065ea1748feac6b2e76613#n77
https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/blob/main/src/libnm-core-impl/nm-setting-ip-config.c#L2828